src/Effects.js

```diff
@@ -1445,6 +1445,35 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.music_wave = {
+      inc: 360/15,
+      heightFactor: 1,
+      heightDivider: 200,
+      yLoc: 300,
+      xTotal: p5.width,
```

Comment:
Nit: Since `xTotal` now just holds the value of `p5.width`, we don't have to save the variable `xTotal` and just replace references to it with p5.width.

Reply:
I believe it is all fixed now. The PR only shows Effects.js as modified. I also made the above change.